packages/flutter/lib/src/material/modal_bottom_sheet.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'package:flutter/material.dart';
+
+import 'debug.dart';
+import 'material.dart';
+import 'material_localizations.dart';
+import 'theme.dart';
+
+/// Shows a modal material design bottom sheet.
+Future<T> showMaterialModalBottomSheet<T>({
```

Comment:
Should we replace the current showModalBottomSheet with this one? Or should I remove this one?